### ISSUE TITLE
Fix markdown lint

### ISSUE THREAD

**Issue by randshell** [sourcerer-io/hall-of-fame#47]
Code: `[![](https://sourcerer.io/fame/$USER/$OWNER/$REPO/images/0)](https://sourcerer.io/fame/$USER/$OWNER/$REPO/links/0)`

Issue: `MD045/no-alt-text: Images should have alternate text (alt text)` markdownlint([MD045](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md045))
